src/server/Makefile.am

```diff
@@ -45,6 +45,10 @@ pbs_server_bin_CPPFLAGS = \
 	@libical_inc@ \
 	@PYTHON_INCLUDES@
 
+if ALPS_ENABLED
+pbs_server_bin_CPPFLAGS += -DMOM_ALPS=1
+endif
+
```

Comment:
We should not be setting the MOM_ALPS flag for compiling the server.  The MOM_ALPS flag should only be used in MoM.

Reply:
We are already compiling the server with enable-alps flag here I am just defining the macro for that.